scripts/build.sh

```diff
@@ -8,6 +8,8 @@ if [ "$SL_BUILD" = true ]; then
   yarn workspace snapshot-listener build
 elif [ "$DL_BUILD" = true ]; then
   yarn workspace discord-bot build
+elif [ "$NO_WEBPACK" = true ]; then
```

Comment:
Since this is `commonwealth` specific, can we move this conditional to be under the `else` branch i.e. conditionally execute `NODE_OPTIONS=--max_old_space_size=4096 yarn workspace commonwealth bundle`.